Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```diff
@@ -70,7 +70,7 @@ internal async Task<List<Result>> SearchAsync(Query query, CancellationToken tok
             if (isEnvironmentVariablePath)
                 locationPath = EnvironmentVariables.TranslateEnvironmentVariablePath(locationPath);
 
-            if (!FilesFolders.LocationExists(FilesFolders.ReturnPreviousDirectoryIfIncompleteString(locationPath)))
+            if (!FilesFolders.ReturnPreviousDirectoryIfIncompleteString(locationPath).IsLocationPathString())
```

Comment:
Reverting this change because if we dont check whether location exists, it will error when we do directory search:
![image](https://user-images.githubusercontent.com/26427004/105626964-b8dcb500-5e87-11eb-86fd-57fcbf00699a.png)

Reply:
Forget this situation. Thank you!